applets/inhibit/inhibit-applet.c

```diff
@@ -244,11 +244,9 @@ gpm_applet_update_tooltip (GpmInhibitApplet *applet)
 static gboolean
 gpm_applet_click_cb (GpmInhibitApplet *applet, GdkEventButton *event)
 {
-	/* react only to left mouse button */
-	if (event->button != 1) {
-		return FALSE;
-	}
-	return gpm_applet_switch_cb (applet);
+	if (gdk_event_triggers_context_menu ((GdkEvent *) event)
```

Comment:
Build failed at Travis-CI
https://app.travis-ci.com/github/mate-desktop/mate-power-manager/jobs/550884848#L2048
```
inhibit-applet.c:247:65: error: expected ')' before 'return'

  247 |         if (gdk_event_triggers_context_menu ((GdkEvent *) event)

      |            ~                                                    ^

      |                                                                 )

  248 |                 return gpm_applet_switch_cb (applet);

      |                 ~~~~~~                                           

inhibit-applet.c:250:1: error: expected expression before '}' token

  250 | }

      | ^
```
Please test building locally for yourself.

Reply:
Duh. Fixed missing parenthesis. Sorry for the noise.